aptos-move/block-executor/src/executor.rs

```diff
@@ -1210,6 +1275,20 @@ where
                             finalized_groups.push((group_key, group_metadata_op, finalized_group));
                         }
 
+                        for (group_key, group_metadata_op) in
+                            output.group_reads_needing_delayed_field_exchange()
+                        {
+                            let finalized_group = unsync_map.finalize_group(&group_key);
+                            if finalized_group.is_empty() != group_metadata_op.is_deletion() {
```

Comment:
ditto re:error condition here: should not be deletion, or empty.

Reply:
Done.